src/Trace/Reporter/ZipkinReporter.php

```diff
@@ -95,98 +95,68 @@ public function report(TracerInterface $tracer)
     }
 
     /**
-     * Convert spans into Zipkin's expected JSON output format.
+     * Convert spans into Zipkin's expected JSON output format. See http://zipkin.io/zipkin-api/#/default/post_spans
+     * for output format.
      *
-     * @param  TracerInterface $tracer
+     * @param TracerInterface $tracer
+     * @param array $headers [optional] HTTP headers to parse. **Defaults to** $_SERVER
      * @return array Representation of the collected trace spans ready for serialization
      */
-    public function convertSpans(TracerInterface $tracer)
+    public function convertSpans(TracerInterface $tracer, $headers = null)
     {
+        $headers = $headers ?: $_SERVER;
         $spans = $tracer->spans();
-        $context = $tracer->context();
-        $traceId = $context->traceId();
+        $rootSpan = $spans[0];
+        $traceId = $tracer->context()->traceId();
+
+        $kindMap = [
```

Comment:
I'm just wondering. Is there any other places that converts those constants into strings? If so, does it make sense to have a helper function for the conversion alongside with those constant definition?